src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1575,7 +1575,7 @@ material.hypochlorous_acid=Hypochlorous Acid
 material.ammonia=Ammonia
 material.hydrofluoric_acid=Hydrofluoric Acid
 material.nitric_oxide=Nitric Oxide
-material.iron_iii_chloride=Iron III Chloride
+material.iron_iii_chloride=Iron(III) Chloride
```

Comment:
```suggestion
material.iron_iii_chloride=Iron (III) Chloride
```

Reply:
I am not sure about the space between Iron and (III) as from quick search most places are using it without it.